src/main/java/org/opentripplanner/index/IndexAPI.java

```diff
@@ -168,7 +168,35 @@ public Response getStop (@PathParam("stopId") String stopIdString) {
            return Response.status(Status.NOT_FOUND).entity(MSG_404).build();
        }
    }
-   
+
+    /**
+     * Helper method for below endpoint to add a ShortStop to a collection of stops with optional extra
```

Comment:
Be more explict and name the 'below endpoint'.

Reply:
Renamed in 163f3429255f5622503a1657a714699fc2a7eb16